samples/stopwatch-coroutines/src/main/java/com/instacart/formula/stopwatch/StopWatchViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.instacart.formula.stopwatch
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.instacart.formula.coroutines.toFlow
+import kotlinx.coroutines.flow.*
```

Comment:
nit: we generally don't use wildcard imports.

Reply:
I agree. I blame Android Studio for this. 😂